toast-tk-runtime/src/main/java/io/toast/tk/runtime/report/TemplateHelper.java

```diff
@@ -10,6 +15,11 @@
 
 public class TemplateHelper {
 
+	private static int Html_Length_Max = 40;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "Html_Length_Max" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)